CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+**Unreleased**
+
+* :meth:`Submission.mod.set_original_content` support 
+  marking a submission as "original content."
+* :meth:`Submission.mod.unset_original_content()` support 
+  unmarking a submission as "original content."
```

Comment:
For the same reasons as above, please change this to
  * :meth:`.unset_original_content` supports
    unmarking a submission as original content.